website/static/js/logTextParser.js

```diff
@@ -243,25 +325,64 @@ var LogPieces = {
     // The new title of node involved
     title_new: {
         view: function (ctrl, logObject) {
-            return returnTextParams('title_new', 'a title', logObject);
+            var url = null;
+            var nodeObject = logObject.embeds.original_node;
+
+            if (paramIsReturned(nodeObject, logObject && nodeObject.data)){
+                 if (nodeObject.data.links && nodeObject.data.attributes) {
```

Comment:
Why is it necessary to check `.data.attributes` here? If you can remove that check, you could use `lodash.get` to get `.data.links.html`.